modules/virt-creating-storage-cluster-fusion-access-san.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/fusion_access_SAN/install-configure-fusion-access-san.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-storage-cluster-fusion-access-san_{context}"]
+= Creating a storage cluster with {IBMFusionSAN}
+
+Once you have installed the {IBMFusionSAN} Operator, you can create a storage cluster with shared storage nodes.
+
+The wizard for creating the storage cluster in the {product-title} web console provides easy to follow steps, and lists the relevant nodes with shared disks.
+
+.Prerequisites
+
+* You have bare-metal worker nodes with visible and attached shared LUNs.
+* You have installed the {IBMFusionSAN} Operator.
+* You have created the `FusionAccess` custom resource (CR) in the `ibm-fusion-access` namespace.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Storage* -> *{IBMFusionSAN}*.
+
+. Click *Create storage cluster*.
+
+. Select the worker nodes that have shared disks.
```

Comment:
I'd make this either "shared LUNs" here or above we switch to "shared disks" just to stay consistent.